shared/fs/row/still-container.js

```diff
@@ -1,62 +1,50 @@
 // @flow
+import * as I from 'immutable'
 import * as FsGen from '../../actions/fs-gen'
 import * as Types from '../../constants/types/fs'
 import * as Constants from '../../constants/fs'
 import {compose, connect, setDisplayName, type TypedState, type Dispatch} from '../../util/container'
 import Still from './still'
 import * as StateMappers from '../utils/state-mappers'
 
-const mapStateToProps = (state: TypedState, {path}) => {
-  const pathItem = state.fs.pathItems.get(path, Constants.unknownPathItem)
+type OwnProps = $Diff<Types.StillRowItem, {rowType: 'still'}> & {
+  routePath: I.List<string>,
+}
+
+const mapStateToProps = (state: TypedState, {path}: OwnProps) => {
+  const _pathItem: Types.PathItem = state.fs.pathItems.get(path, Constants.unknownPathItem)
   const _username = state.config.username || undefined
   const _downloads = state.fs.downloads
   return {
-    _username,
     _downloads,
-    path,
-    kbfsEnabled: StateMappers.mapStateToKBFSEnabled(state),
-    pathItem,
+    _kbfsEnabled: StateMappers.mapStateToKBFSEnabled(state),
```

Comment:
This whole `state-mappers` file doesn't seem useful to me. It just has one function and it's not named in a way that is intuitive. This util function probably belongs in `constants/fs.js`.

Reply:
I just copied them from the old code. I'm OK with getting rid of this or organize them in a different way, but didn't want to address too much initially. I can try to clean these up in a follow-up commit to this branch.